app/Rules/ValidTestQuestionsTotalScore.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+
+class ValidTestQuestionsTotalScore implements Rule
+{
+    /**
+     * Create a new rule instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        $score = 0;
+
+        foreach ($value as $question) {
+            if (isset($question['score'])) {
+                $score += $question['score'];
+            }
```

Comment:
What about this? 

```suggestion
            $score += $question['score'] ?? 0;
```

Reply:
https://www.php.net/manual/ru/migration70.new-features.php#migration70.new-features.null-coalesce-op

Reply:
https://stackoverflow.com/questions/1276909/php-syntax-question-what-does-the-question-mark-and-colon-mean